### ISSUE TITLE
VFDB databases are missing

### ISSUE THREAD

**Issue by sallybyers** [gbouras13/pharokka#291]
* pharokka version: v1.4.1
* Python version: Python 3.11.4
* Operating System: MacOS

Hi George,

I downloaded pharokka today and tried to run the program using: 

pharokka.py -i x.fasta -o x_pharokka -d /Users/.../pharokka/pharokka_v1.4.0_databases -t 1 

The outputs are: 

2023-09-05 16:07:27.016 | INFO     | __main__:main:84 - Starting Pharokka v1.4.1
2023-09-05 16:07:27.017 | INFO     | __main__:main:85 - Command executed: Namespace(infile='x.fasta', outdir='x_pharokka', database='/Users/sallybyers/pharokka/pharokka_v1.4.0_databases', threads='1', force=False, prefix='Default', locustag='Default', gene_predictor='phanotate', meta=False, split=False, coding_table='11', evalue='1E-05', fast=False, mmseqs2_only=False, meta_hmm=False, dnaapler=False, custom_hmm='', genbank=False, terminase=False, terminase_strand='nothing', terminase_start='nothing', citation=False)
2023-09-05 16:07:27.017 | INFO     | __main__:main:86 - Repository homepage is https://github.com/gbouras13/pharokka
2023-09-05 16:07:27.017 | INFO     | __main__:main:87 - Written by George Bouras: george.bouras@adelaide.edu.au
2023-09-05 16:07:27.017 | INFO     | __main__:main:89 - Checking database installation in /Users/sallybyers/pharokka/pharokka_v1.4.0_databases.
2023-09-05 16:07:27.017 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2023-09-05 16:07:27.017 | ERROR    | __main__:main:94 - The database directory was unsuccessfully checked. Please run install_databases.py.

I ran install_databases.py and downloaded all of the databases again, but am running into the same issue with the VFDB Databases being missing. 

Cheers,

Sally

**Comment by gbouras13** [comment]
Hi Sally, can you run `ls` on the pharokka_v1.4.0_databases directory?

George

**Comment by sallybyers** [comment]
Hey George,

Thanks for the really quick response! The ls output is:

1Aug2023_data.tsv
1Aug2023_genomes.fa.msh
CARD
CARD.dbtype
CARD.index
CARD.lookup
CARD.source
CARD_h
CARD_h.dbtype
CARD_h.index
VFDB_setB_pro.fas
all_phrogs.h3m
aro_index.tsv
phrog_annot_v4.tsv
phrogs_db
phrogs_db.dbtype
phrogs_db.index
phrogs_profile_db
phrogs_profile_db.dbtype
phrogs_profile_db.index
phrogs_profile_db_consensus
phrogs_profile_db_consensus.dbtype
phrogs_profile_db_consensus.index
phrogs_profile_db_h
phrogs_profile_db_h.index
phrogs_profile_db_seq
phrogs_profile_db_seq.dbtype
phrogs_profile_db_seq.index
phrogs_profile_db_seq_h
phrogs_profile_db_seq_h.index
vfdb
vfdb.dbtype
vfdb.index
vfdb.lookup
vfdb.source
vfdb_h
vfdb_h.dbtype
vfdb_h.index

Cheers,

Sally

**Comment by gbouras13** [comment]
Ok so that is pretty strange.

Can you try manually downloading it from this link?

https://zenodo.org/record/8276347/

The files you expect to see are:

```
VFDB_DB_NAMES = [
    "vfdb",
    "vfdb.dbtype",
    "vfdb.index",
    "vfdb.lookup",
    "vfdb.source",
    "vfdb_h",
    "vfdb_h.dbtype",
    "vfdb_h.index",
    "VFDBclusterRes_cluster.tsv",
    "VFDBclusterRes_rep_seq.fasta",
]
```

George

**Comment by gbouras13** [comment]
Hi Sally,

I've just tried downloading the databases again (it took a while, perks of slow internet in Australia :) ).

I've checked that I do have the required files in my database, plus Pharokka now has continuously integrated testing so that would've picked up the issue. Therefore, I am fairly sure it is an issue with your set-up.

I'd first try downloading again

```
install_databases.py -o hopefully_this_fixes_the_issue_directory
```

If that doesn't work then let me know, it would probably best to send an email and I will help out some more 1-1.

George
